tf2onnx/onnx_opset/nn.py

```diff
@@ -1407,11 +1407,11 @@ def version_11(cls, ctx, node, **kwargs):
         nearest_mode = "floor"
         if "align_corners" in node.attr and node.attr["align_corners"].i:
             transformation_mode = "align_corners"
+            nearest_mode = "round_prefer_ceil"
         if "half_pixel_centers" in node.attr and node.attr["half_pixel_centers"].i:
             if node.type == "ResizeNearestNeighbor" and not ctx.is_target(constants.TARGET_TENSORRT):
                 # TensorRT only supports nearest_mode = "floor" for mode = "nearest"
-                transformation_mode = "half_pixel"
-                nearest_mode = "round_prefer_ceil"
+                transformation_mode = "tf_half_pixel_for_nn"
```

Comment:
Is this for debug?

Reply:
Which name? tf_half_pixel_for_nn? There is no name for debug. All of them are valid attributes of onnx Resize op.
https://github.com/onnx/onnx/blob/main/docs/Changelog.md#resize-11